WebRoot/css/zm.css

```diff
@@ -7147,6 +7143,28 @@ a.ZmLinkDisabled {
 	@FullWidth@
 }
 
+/* Add External calendar dialog */
```

Comment:
Minor: alignment is bit different across different styles.

Reply:
Fixed in https://github.com/Zimbra/zm-web-client/pull/140/commits/c476e679237602e6d9249ab741066eb924ddcdd0